.buildkite-external-version

```diff
@@ -1 +1 @@
-main
+jb/trim_ci_job
```

Comment:
Should be on main for the PR merge and then a separate merge to main in the PR repo